src/terms/xc.jl

```diff
@@ -6,13 +6,22 @@ Exchange-correlation term, defined by a list of functionals and usually evaluate
 """
 struct Xc
     functionals::Vector{Symbol}  # Symbols of the functionals (Libxc.jl / libxc convention)
-    scaling_factor::Real         # to scale by an arbitrary factor (useful for exploration)
+    scaling_factor::Real         # Scales by an arbitrary factor (useful for exploration)
+
+    # Density cutoff for XC computation: Below this value gridpoint counts as zero
+    # `nothing` implies that Libxc defaults are used (different for every functional)
+    density_threshold::Union{Nothing,Float64}
```

Comment:
can you quickly say what the default looks like? Is it 0 or a non-zero value?

Reply:
It's not zero but super close (1e-34, 1e-24, ...) I'm not sure where the values come from. Could be some numerical saveguard against NaNs I suppose?

Reply:
OK so numerical zero basically, but no "cheating" (ie it's not 1e-4 or what)

Reply:
Indeed. It looked like completely reasonable default values, but where raising them a bit could improve performance.